packages/performance-tests/test/setup.ts

```diff
@@ -2,5 +2,5 @@ import dotenv from "dotenv";
 
 const { error } = dotenv.config();
 
-if (error)
-  throw new Error("no env file found");
+if (error && process.env.CI !== "1")
```

Comment:
It's generally better to just check for non emptiness of an env variable than a specific value 
```ts
if (error && !process.env.CI)
```

Reply:
done. Didn't realize that would produce a boolean. good to know thanks

Reply:
it technically does but there are some caveats which I'll explain in a dm